src/aat/resources/features/F-107/F-107.feature

```diff
@@ -0,0 +1,53 @@
+@F-107
+Feature: F-107: Organisation Policies on Cases
+
+    Background: Load test data for the scenario
+    Given an appropriate test context as detailed in the test data source
+
+    @S-925
+    Scenario: must successfully create a case with a case type containing some OrganisationPolicy fields
+      Given a user with [an active profile in CCD]
+      And a successful call [to create a token for case creation] as in [F-107_Case_Data_Create_Token_Creation]
+      When a request is prepared with appropriate values
+      And the request [contains some OrganisationPolicy fields with all correct values]
+      And it is submitted to call the [Submit Case Creation as Caseworker] operation of [CCD Data Store]
+      Then a positive response is received
+      And the response has all other details as expected
+
```

Comment:
More ideally, we could probably apply a verification step to get the details of the case just created, using a ste to call a relevant api operation. Not a must, though, for just practical purposes.

Reply:
👍